subsys/mgmt/mcumgr/mgmt/Kconfig

```diff
@@ -25,3 +25,10 @@ config MCUMGR_MGMT_NOTIFICATION_HOOKS
 	  or decline the current operation, by returning false this will signal to the calling
 	  function that the request should be denied, for informal-only notifications or
 	  acceptable, true must be returned by all the registered notification handlers.
+
+config MCUMGR_MGMT_CUSTOM_PAYLOAD
+	bool "MCUmgr custom payload"
+	help
+	  When this config is enabled, the generation of the start- and end byte in
+	  smp_handle_single_payload will be skipped which allows a user to add fully custom
+	  user defined payload in the SMP messages.
```

Comment:
This description needs to be updated to mention that `custom_payload` must be set if a user wants to define their own format, the current wording reads like it is enabled for all handlers just by enabling this option

Reply:
What do you think about the updated description?